src/service/data-service/search.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+class SearchService {
+  constructor(offers) {
```

Comment:
```suggestion
  constructor(articles) {
```
много где так есть. пройдись по проекту, и замени везде